user/languages/csharp.md

```diff
@@ -95,12 +95,19 @@ By default, Travis CI does not test against .NET Core. To test against .NET Core
 language: csharp
 mono: none
 dotnet: 1.0.1
+dist: trusty
+sudo: required
```

Comment:
I don't believe `sudo` is required. I've just set-up a .NET Core build, and it builds fine configured with just the trusty distro. See below:

https://travis-ci.org/JSkimming/Castle.Core.AsyncInterceptor/jobs/221394769/config

This may be working because it's running on the [Container-based Ubuntu Trusty in Public Beta](https://blog.travis-ci.com/2016-11-08-trusty-container-public-beta "Travis CI Blog").